cookbook/bundles/dependencies.rst

```diff
@@ -0,0 +1,51 @@
+.. index::
+   single: Bundle; Dependencies
+
+How to Use Bundle Dependencies to load other Bundles
+====================================================
+
+When working on your own bundle(s), you'll sometimes have the need to reuse
+other bundles, either by just requiring, overriding or inherit from them.
+
+While Composer takes care about letting you make sure these dependencies are
+loaded, you'll also need to enable these bundles in the Kernel.
+
+If your bundle is meant for reuse, then this will typically complicate your
+installation documentation, and if your dependencies change over time they
+will likewise complicate your upgrade documentation, and make upgrading
+more tedious for your users.
```

Comment:
good point

what about _"If your bundle is meant to be reused, bundle dependencies will complicate your installation documentation. This makes installation and upgrading your bundle more tedious for your users."_